### ISSUE TITLE
[Fix] sys.stderr.fileno() throws an AttributeError exception

### ISSUE THREAD

**Issue by Evilran** [microsoft/playwright-python#246]
When I tried to delivered the playwright tasks via `celery==4.4.7`, I got an `AttributeError` exception at line 50 in `main.py`. 

`except AttributeError:`

I've checked the source from `pytest`, the original code is:

 ```
     except (AttributeError, io.UnsupportedOperation):
            # pytest-xdist monkeypatches sys.stderr with an object that is not an actual file.
            # https://docs.python.org/3/library/faulthandler.html#issue-with-file-descriptors
            # This is potentially dangerous, but the best we can do.
            return sys.__stderr__.fileno()
```

It perfectly works if I changed it to the original code, so I think we should be consistent with the original source code to enhance compatibility.

I also checked the celery official docs. The reason why I got an `AttributeError` exception probably is that the log proxy of celery is not support the `stderr` attribute.

**Comment by mxschmitt** [comment]
@Evilran did you try it locally after adding `AttributeError` to your local `playwright/main.py`?

Sounds good to me that we can add it.

**Comment by Evilran** [comment]
@mxschmitt I did that before and it works as expected! I think it would be nice if you could change it in the next stable version of `playwright-python` package 👍

**Comment by mxschmitt** [comment]
> @mxschmitt I did that before and it works as expected! I think it would be nice if you could change it in the next stable version of `playwright-python` package 👍

Cool thanks for confirming, will adjust it.

**Comment by mxschmitt** [comment]
@Evilran released version 0.151.1 https://pypi.org/project/playwright/0.151.1/

**Comment by Evilran** [comment]
> @Evilran released version 0.151.1 https://pypi.org/project/playwright/0.151.1/

Awesome work, cool!

**Comment by yswtrue** [comment]
it seems not to work in 0.171.0 and celery 4.47
